### PR TITLE
Add clarification about Braintree Settlement Report

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) adds clarification about Braintree Settlement Report

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/reports/sales-reports.html

### CODE REVIEW

File: src/reports/sales-reports.md [hunk 1]
```diff
@@ -52,3 +52,10 @@ The [Coupons Report]({% link marketing/price-rules-cart-coupon-report.md %}) can
 *PayPal Settlement Report*
 
 The [PayPal Settlement Report]({% link payment/paypal-settlement-reports.md %}) can be filtered by date, merchant account, transaction ID, invoice ID, or PayPal reference ID. The report includes the type of event, such as a debit card transaction, the start and finish dates, gross amount, and related fees. The report can be automatically updated with the most current data from PayPal.
+
+## Braintree Settlement
+
+![]({% link images/images/reports-sales-braintree-settlement.png %}){: .zoom}
+*Braintree Settlement Report*
+
+The [Braintree]({% link payment/braintree.md %}) Settlement Report can be filtered according to creation date, amount, status, transaction type, payment type, transaction ID, order ID, PayPal payment ID, type, merchant account ID or settlement batch ID. The report contains the transaction ID, order ID, PayPal payment ID, type, creation date, amount, settlement code, status, settlement response text, reimbursement IDs, merchant account ID, settlement batch ID and the currency used.
```
Comment (erikmarr) [inline src/reports/sales-reports.md:61]:
```suggestion
The [Braintree]({% link payment/braintree.md %}) Settlement Report can be filtered according to creation date, amount, status, transaction type, payment type, transaction ID, order ID, PayPal payment ID, type, merchant account ID or settlement batch ID. The report contains the transaction ID, order ID, PayPal payment ID, type, creation date, amount, settlement code, status, settlement response text, reimbursement IDs, merchant account ID, settlement batch ID, and the currency used.
```

**Review by erikmarr — CHANGES_REQUESTED** [review]

**Comment by erikmarr** [comment]
@magento import code to magento-devdocs/merchdocs

**Review by erikmarr — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
@erikmarr the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1321`.

**Comment by erikmarr** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@erikmarr the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1321`.

**Pull request merged**